emsdk.py

```diff
@@ -443,9 +434,9 @@ def win_set_environment_variables(env_vars_to_add, system):
     errlog('SendMessageTimeout failed with error: ' + str(e))
 
 
-def win_delete_environment_variable(key, system=True):
+def win_delete_environment_variable(key, system=True, user=True):
```

Comment:
Ditto remove the defaults.

Reply:
This function is not used anywhere, so we can remove the whole thing. I was planning to do a clean up after this PR.